app/src/main/java/org/mozilla/rocket/privately/PrivateModeActivity.kt

```diff
@@ -206,4 +234,46 @@ class PrivateModeActivity : BaseActivity(),
         }
         return false
     }
+
+    private fun initBroadcastReceivers() {
+        uiMessageReceiver = object : BroadcastReceiver() {
+            override fun onReceive(context: Context, intent: Intent) {
+                if (intent.action == Constants.ACTION_NOTIFY_RELOCATE_FINISH) {
+                        showOpenSnackBar(intent.getLongExtra(Constants.EXTRA_ROW_ID, -1))
+                }
+            }
+        }
+    }
+
+    // TODO consider to combine with MainActivity.showOpenSnackBar()
```

Comment:
Could you do it in this PR? Otherwise we'll have two copy of the code with the same logic

Reply:
Move to ViewUtils.